frontend/src/components/home-page/components/podcast.tsx

```diff
@@ -0,0 +1,23 @@
+import styles from '../styles.module.scss';
```

Comment:
Podcast card component should be moved out to the common directory, as it may be used somewhere else